### PR TITLE
Doc updates for typos and clearification

### PR DESCRIPTION
**Pull request by ahpook**
A few miscellaneous doc updates fixing things I ran into today.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ 🚫n/a ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ✔️  ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ 🤷‍♂️ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

### CODE REVIEW

File: docs/tasks.md [hunk 1]
```diff
@@ -370,7 +370,7 @@ steps:
 ```
 
 
-### Variable Subtituation
+### Variable Substitution
```
Comment (bobcatfish) [inline docs/tasks.md]: oh boy 😅

**Comment by bobcatfish** [comment]
hm that cat is not space related at all

**Comment by bobcatfish** [comment]
Thanks @ahpook !! :D

**Comment by bobcatfish** [comment]
Oh I guess we need that CLA business first 😇

**Comment by ahpook** [comment]
@googlebot i signed it

**Pull request merged**